docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/variants/connect/entrypoints/Connect.scss

```diff
@@ -1224,7 +1227,7 @@ form button[type=submit]:hover {
   .section-nav .level-1.contains-current.has-children>div a:active,
   .section-nav .level-1.contains-current.has-children>div a:focus {
     //background: $section-menu-mobile-tablet-family-active-background;
-    //color: $section-menu-mobile-tablet-family-active-color;
+    color: $section-menu-mobile-tablet-family-active-color;
```

Comment:
Please remove code rather than commenting it out. Commented out code becomes "noise" when you're reading the file or worse, trying to do a search.